### PR TITLE
📝 Adding A Project-Bootup-Flow Markdown

### PR DESCRIPTION
**Pull request by taljacob2**
Adding a markdown file that shows visually (with a
[mermaid] graph) the flow of the bootup process
of the project once the client runs the `vagrant up`
command.

### You can view this file on my branch [here](https://github.com/taljacob2/JobSeeker/blob/doc/adding-bootup-flowchart/docs/Project-Bootup-Flow.md).

[mermaid]: https://mermaid-js.github.io/

Signed-off-by: Tal Jacob <taljacob2@gmail.com>

### CODE REVIEW

**Comment by taljacob2** [comment]
You can view this file on my branch [here](https://github.com/taljacob2/JobSeeker/blob/doc/adding-bootup-flowchart/docs/Project-Bootup-Flow.md).

File: docs/Project-Bootup-Flow.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Project Boot-Up Flow
+
+These graphs show visually the flow of the bootup process of the project once the client runs the `vagrant up` command.
```
Comment (rrasouli) [inline docs/Project-Bootup-Flow.md:3]: Grammar, swap show visually to 'visually show'

**Review by paOmer — APPROVED** [review]
LGTM

File: docs/Project-Bootup-Flow.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- ## Boot-Up Structure
+    ```mermaid
+    flowchart TB
+    subgraph Vagrant["Vagrant (With 'fedora/34-cloud-base' Running On VirtualBox)"]
```
Comment (Yarboa) [inline docs/Project-Bootup-Flow.md:8]: > Running On VirtualBox

Consider this Virtual Machine running in VirtualBox

Comment (taljacob2) [reply docs/Project-Bootup-Flow.md]: This was updated in [aa55a5e](https://github.com/redhat-beyond/JobSeeker/commit/aa55a5e1b3ca36d88ae12d308465cf129dc85464). You may review again.

File: docs/Project-Bootup-Flow.md (hunk 1 above)
Comment (taljacob2) [reply docs/Project-Bootup-Flow.md]: This was updated in [aa55a5e](https://github.com/redhat-beyond/JobSeeker/commit/aa55a5e1b3ca36d88ae12d308465cf129dc85464). You may review again.

**Review by Yarboa — COMMENTED** [review]
@paOmer @ShirleyFichman @DeanBiton @amittcohen

Please review

**Review by paOmer — APPROVED** [review]
LGTM

**Review by amittcohen — APPROVED** [review]
LGTM

**Review by ShirleyFichman — APPROVED** [review]
LGTM

**Pull request merged**